config/admin_routing.yaml

```diff
@@ -15,3 +15,13 @@ sylius_price_history_admin_channel_pricing_log_entry_index:
                 header: sylius_price_history.ui.channel_pricing_history
                 subheader: sylius_price_history.ui.show_history_for_channel_pricing
                 product_variant: "expr:service('sylius.repository.product_variant').find($variantId)"
+
+sylius_admin_product_show:
+    path: /products/{id}
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.product::showAction
+        _sylius:
+            section: admin
+            permission: true
+            template: "@SyliusPriceHistoryPlugin/Admin/Product/show.html.twig"
```

Comment:
Couldn't you just override the `_simpleProduct` template instead?

Reply:
As we determined this solution did not work

Reply:
What was the problem? Because it should work, only you would have to copy the given template to the end application